entry.go

```diff
@@ -208,13 +181,67 @@ func (te *TunaEntry) StartReverse(stream *smux.Stream) error {
 			return
 		}
 		stream.Close()
+
+		lastComputed := common.Fixed64(0)
+		lastClaimed := common.Fixed64(0)
+		lastUpdate := time.Now()
+		claimInterval := time.Duration(te.Config.ReverseClaimInterval) * time.Second
+		onErr := nkn.NewOnError(1, nil)
+		isClosed := false
+
+		npc, err := te.Wallet.NewNanoPayClaimer(int32(claimInterval/time.Millisecond), onErr, te.Config.ReverseBeneficiaryAddr)
+		if err != nil {
+			log.Println(err)
+			return
+		}
+
+		go func() {
+			for {
+				err := <-onErr.C
+				if err != nil {
+					log.Println("Couldn't claim nano pay:", err)
+					if npc.IsClosed() {
+						break
```

Comment:
You shouldn't break without close session and set isClosed to true because this npc will never be able to claim any nanopay updates, which means entry will not get any rewards